src/runtime/pybuffer.cs

```diff
@@ -0,0 +1,228 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Python.Runtime
+{
+    public class PyBuffer : IDisposable
```

Comment:
I'd make this class `sealed`, since it does not provide any overridable behavior.